src-docs/src/views/overlay_mask/overlay_mask_example.js

```diff
@@ -46,16 +46,57 @@ export const OverlayMaskExample = {
             to make before choosing to use an overlay. At the very least, you
             must provide a visible button to close the overlay. You can also
             pass an <EuiCode>onClick</EuiCode> handler to handle closing the
-            overlay. However, be wary of using <EuiCode>onClick</EuiCode> and{' '}
-            <EuiCode>children</EuiCode>, as clicking the{' '}
-            <EuiCode>children</EuiCode> will also trigger the{' '}
-            <EuiCode>onClick</EuiCode>.
+            overlay.
           </p>
         </div>
       ),
       props: { EuiOverlayMask },
-      snippet: overlayMaskSnippet,
+      snippet: `<EuiOverlayMask onClick={() => {}}>
+  <!-- Content goes here -->
+</EuiOverlayMask>`,
       demo: <OverlayMask />,
     },
+    {
+      title: 'Masks with header',
+      source: [
+        {
+          type: GuideSectionTypes.JS,
+          code: overlayMaskHeaderSource,
+        },
+        {
+          type: GuideSectionTypes.HTML,
+          code: overlayMaskHeaderHtml,
+        },
+      ],
+      text: (
+        <div>
+          <p>
+            Managing z-index levels of multiple portalled content and their
```

Comment:
```suggestion
            Managing z-index levels of multiple portal-positioned components and their
```

Maybe?
"portalled content" seemed unclear